PrimaryInstallScript.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+if [ $EUID -ne 0 ]; then
+	echo "Please run as root user"
+	exit 1
+fi
+
+ping -q -c1 google.com > /dev/null
+ 
+if [ $? -ne 0 ]; then
+	echo "No Internet Connection...Aborting"
+	exit 1
+fi
+
+#install Epsilon-Hermes
+git clone https://github.com/UCSolarCarTeam/Epsilon-Hermes.git /home/pi/Epsilon-Hermes/
+(cd /home/pi/Epsilon-Hermes && cd ../ && mv ./Epsilon-Hermes ./src
+&& mkdir Epsilon-Hermes
```

Comment:
Syntax error, I think you need `\` before the new line, since commands don't start with `&&`
`cd /home/bill/Epsilon-Hermes && cd ../ && mv ./Epsilon-Hermes ./src \`